target/linux/ath79/mikrotik/base-files/lib/upgrade/platform.sh

```diff
@@ -31,6 +31,7 @@ platform_do_upgrade() {
 	local board=$(board_name)
 
 	case "$board" in
+	mikrotik,routerboard-493|\
```

Comment:
no "ah" suffix

Reply:
I wonder if a wildcard is appropriate here, as in `mikrotik,routerboard-4*` as all these 4xx series boards need the same

Reply:
> no "ah" suffix

Strangely, I found that sysupgrade rejected a version I build using routerboard-493ah. It seemed to expect routerboard-493, so I changed things to reflect that. I will take another look at this to be sure.

Reply:
that's because at line 7 of your DTS you have the wrong string.

Reply:
> I wonder if a wildcard is appropriate here, as in mikrotik,routerboard-4* as all these 4xx series boards need the same

Please, no wildcards in base-files. It's really terrible if you want to find the settings of a particular device afterwards (you cannot grep reliably anymore).

Reply:
@f00b4r0, where would I find the right string? (Note that I ended up using "mikrotik,routerboard-493" because sysupgrade rejected "mikrotik,routerboard-493ah."

Reply:
Well, your old version has the old incorrect compatible "mikrotik,routerboard-493" in the DTS, and that's used for "idenfication" in the upgrade process. So, you would have to update all occurences of that string in your current PR. Then, you would need to force upgrade from your old "incorrect" version, as the new corrected one does not now the old "incorrect" string anymore.

Reply:
@flyn-org as I pointed out above DTS compatible must agree with this script case. Fix both at the same time to use « ah » and it’ll work.

Reply:
Okay. My confusion came from the fact that a booted the board using an old firmware before I tried to run `sysupgrade` for the first time. That firmware must have used a string different than the one I used first, routerboard-493ah.